.github/workflows/python.yaml

```
@@ -106,6 +106,8 @@ jobs:
         run: |
           echo '${{ toJson(secrets) }}' \
             | jq -r 'to_entries[] | "\(.key)=\(.value)"' > .devcontainer/.env
+          echo '${{ toJson(env) }}' \
+            | jq -r 'to_entries[] | "\(.key)=\(.value)"' > .devcontainer/.env
```

You need to append, otherwise you'll overwrite the secrets.

```suggestion
            | jq -r 'to_entries[] | "\(.key)=\(.value)"' >> .devcontainer/.env
```